src/coreclr/jit/emitarm64.cpp

```diff
@@ -6932,8 +7001,18 @@ void emitter::emitIns_R_R_R_I(instruction ins,
         }
     }
 
-    dispIns(id);
-    appendToCurIG(id);
+    // Now the instruction is EITHER emitted OR used to overwrite the previously-emitted instruction.
+    if (reuseInstr == nullptr)
+    {
+        // Then this is the standard exit path and the instruction is to be appended to the instruction group.
+        dispIns(id);
+        appendToCurIG(id);
+    }
+    else
+    {
+        // The instruction is copied over the last emitted insdtruction.
+        memcpy(reuseInstr, id, sizeof(tempInstrDesc));
```

Comment:
In the current code, does the new instruction get output to the JitDisasm/JitDump?

In my suggested mode, we would have already output the instruction. The "backup" function would need to output clearly that the function being removed has been removed.

Reply:
I have implemented your "backup" function.